assignment06 - Copy/index.php

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Assignment 06</title>
+<meta charset="utf-8">
+</head>
+<body>
+<!--Deepa NAtarajan
+    assignment06.php
+    19WI_INFO_2340_WW - Online - Internet Scripting
+    Instructor: Ms.Lisa Thoendel
+    -->
+
+<h1>Hello World</h1>
+<div> Allow me to introduce myself! My name is<?php echo " Deepa Natarajan!"; ?></div>
```

Comment:
great php statements! complete with quotation marks and those pesky semicolons! 👍